app/Services/PolygonService.php

```diff
@@ -323,4 +347,59 @@ protected function transformAndStorePoints($geojson, $sitePolygonProperties): ar
 
         return $this->createGeojsonModels($polygonsGeojson, $sitePolygonProperties);
     }
+
+    public function isFieldInvalid($field, $value)
+    {
+        if (is_null($value) || $value === '') {
+            return true;
+        }
+
+        switch ($field) {
+            case 'plantstart':
+                return ! $this->isValidDate($value);
```

Comment:
If you take a close look, I don't think you are following the same method naming.
you have `isSomethingValid` vs `isValidSomething`. 
My suggestion is to have `isInvalidField`, but do not mix

Reply:
Done.